tar.cabal

```diff
@@ -49,7 +49,7 @@ library
 library tar-internal
   default-language: Haskell2010
   build-depends: base       >= 4.11  && < 5,
-                 array                 < 0.6,
+                 array      >= 0.5.6 && < 0.6,
```

Comment:
Unfortunately `array` is a transitive dependency of `template-haskell` and as such is not reinstallable. So demanding `>= 0.5.6` is equivalent to demanding GHC >= 9.6, which is too extreme.